definitions/tools/sequence_align_and_tag.cwl

```diff
@@ -9,11 +9,13 @@ requirements:
     - class: SchemaDefRequirement
       types:
           - $import: ../types/sequence_data.yml
+          - $import: ../types/trimming_options.yml
     - class: ResourceRequirement
       coresMin: 8
       ramMin: 20000
+    - class: InlineJavascriptRequirement
     - class: DockerRequirement
-      dockerPull: "mgibio/alignment_helper-cwl:1.0.0"
+      dockerPull: "mgibio/alignment_helper-cwl:1.1.0"
```

Comment:
Assume this version of image will have latest flexbar installed and a PR will be created for it.

Reply:
It was added in genome/docker-alignment_helper-cwl#1.  I created the new release branch from master after that PR was merged.

Reply:
👍  You are right. I didn't check it.